frontend/jest.config.js

```diff
@@ -5,8 +5,10 @@ module.exports = {
   moduleNameMapper: {
     '^.+\\.(css|scss)$': 'identity-obj-proxy',
   },
+  transformIgnorePatterns: [
```

Comment:
Do we need this here?
Can't we simple only do 
`// Require the default configuration.
module.exports = require('@shopgate/pwa-unit-test/jest.config');`
in this file?

Reply:
Yup your right. I tested and updated.